sdk/common/src/main/java/com/mesosphere/sdk/offer/taskdata/EnvUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.mesosphere.sdk.offer.taskdata;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.mesos.Protos.Environment;
+
+/**
+ * Utilities relating to environment variable manipulation
+ * TODO(nickbp): Remove in favor of boilerplated env access
```

Comment:
This was just intended as a temporary note. There's a separate branch based on this one which does similar work for envvar handling, to be put up as a separate PR once this one's merged.